next.config.js

```diff
@@ -97,21 +98,29 @@ const configVals = {
                 ],
             },
             {
-                //https://github.com/nextauthjs/next-auth/issues/2408
-                source: '/api/auth/:slug',
+                source: '/(.*).(jpg|jpeg|png|gif|svg|ico|woff2|ttf)', // all images and fonts
                 headers: [
                     {
                         key: 'Cache-Control',
-                        value: 'no-store, max-age=0',
+                        value: 'public, max-age=31536000, immutable',
```

Comment:
How does this header interact with the `minimumCacheTTL` value? The docs mention that `next/image` will use whichever value between the two is greater, so is this always overriding `minimumCacheTTL`?

Reply:
According to the docs, if `max-age` is greater than `minimumCacheTTL`, then `max-age` is favored. However, most of the `/_next/image` paths do not end with .jpg, .png (etc). Rather it ends up being a long query string `.../_next/image/?url=https%3A%2F%2Fi.ytimg.com%2Fvi%2Fah7BVaQ47n4%2Fsddefault.jpg&w=3840&q=75` and so the header regex doesn't match this. In these cases, will cache for 24 hours at minimum.